knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -475,6 +475,20 @@ func updateEventingKafka(kafkachannel serverlessoperatorv1alpha1.Channel) mf.Tra
 // setBootstrapServers sets Kafka bootstrapServers value in kafka-broker-config
 func configureKafkaBroker(kafkaBrokerDefaultConfig serverlessoperatorv1alpha1.BrokerDefaultConfig) mf.Transformer {
 	return func(u *unstructured.Unstructured) error {
+		// patch the deployment and enable the relevant controllers
+		if u.GetKind() == "Deployment" && u.GetName() == "kafka-controller" {
+
+			var deployment = &appsv1.Deployment{}
+			if err := scheme.Scheme.Convert(u, deployment, nil); err != nil {
+				return err
+			}
+
+			// we just keep the sink controller disabled
+			deployment.Spec.Template.Spec.Containers[0].Args = []string{"--disable-controllers=sink-controller"}
```

Comment:
Shouldn't this be based on the sink or broker flags?

Reply:
yes, but we have no sink flag yet - that will be on the follow up PR.

Now, when once install from master: all is installed. 

this PR changes that, to only enalbe the broker/trigger (and keep the sink disabled).

The follow up PR, will tweak around this section, but I want to land a little better separation (as I now do get the sink, which I do not want)

Reply:
OK

/lgtm